mp/src/game/client/glow_outline_effect.cpp

```diff
@@ -12,13 +12,35 @@
 #include "materialsystem/itexture.h"
 #include "view_shared.h"
 #include "viewpostprocess.h"
+#ifdef GLOWS_ENABLE
+#ifdef NEO
+#include "neo_gamerules.h"
```

Comment:
NIT: `#if defined NEO && defined GLOWS_ENABLED`? Would look nicer imo but ignore if we have a standard I am unaware of :^)